API/BookSharing/BookSharing.Infrastructure/BookApi/BookApiRetriver.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading.Tasks;
+using BookSharing.Infrastructure.Interface;
+using Microsoft.Extensions.Logging;
+
+namespace BookSharing.Infrastructure.BookApi
+{
+    public class BookApiRetriver : IBookInformationFromExternalSource
+    {
+        private readonly IGoogleBookApiClient _bookApi;
+        private readonly ILogger<BookApiRetriver> _logger;
+        public BookApiRetriver(IGoogleBookApiClient bookApi, ILogger<BookApiRetriver> logger)
+        {
+            this._bookApi = bookApi;
```

Comment:
nie uzywajmy this, albo podłoga (_) albo this. przyjmijmy ze w tym projekcie bedzie to podłoga